docs/source/contribution.rst

```diff
@@ -431,15 +431,22 @@ To build the documentation, you need to install `Sphinx <http://www.sphinx-doc.o
 
   $ pip install sphinx sphinx_rtd_theme
 
-Then you can build the documentation in HTML format locally::
+Docstrings (documentation comments in the source code) are collected from the installed Chainer module.
+If Chainer has not been installed yet, then ``make html`` will yield an error that the reference sources cannot
+be read, such as ``chainer.AbstractSerializer``.
+
+Alternatively, if you have modified docstrings, make sure to install the module before building the
```

Comment:
>Alternatively,

Is this needed?
It sounds like there are two methods that are explaned. What are those?

Reply:
Both situations have the same remedy--install from top of GitHub directory. I edited to provide clarity.